src/index.html

```diff
@@ -5,8 +5,414 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home"
+        class="header__icon">
+        <img
+          src="./images/logo.png"
+        alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Laptops & Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#Gadgets" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <div class="container">
+        <div class="card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+
+          <div class="card__content">
+            <h3 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+
+            <p class="card__code">
+              Product code: 195434
+            </p>
+
+            <div class="card__raiting">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__raiting-review">
+                Reviews: 5
+              </p>
+
+            </div>
+
+            <p class="card__price">
+              Price:
+              <span class="card__price-bold">$2,199</span>
+            </p>
+
+            <a href="#"
+              class="card__buy"
+              data-qa="card-hover"
+            >
```

Comment:
Need to check and fix it everywhere
```suggestion
            <a
              href="#"
              class="card__buy"
              data-qa="card-hover"
            >
```